### PR TITLE
fix(ctb): MIPS.sol Inline Documentation Refinement

### PR DESCRIPTION
**Pull request by refcell**
**Description**

Update MIPS.sol docs to be Inline with FPVM Specs.

### CODE REVIEW

File: packages/contracts-bedrock/contracts/cannon/MIPS.sol [hunk 1]
```diff
@@ -172,25 +184,38 @@ contract MIPS {
...
                 v0 = a2;
-            } else {
+            }
+            else {
                 v0 = 0xFFffFFff;
                 v1 = EBADF;
             }
-        } else if (syscall_no == 4004) { // write
+        }
+        // write: Similar behavior as Linux/MIPS with support for unaligned writes.
```
Comment (protolambda) [inline packages/contracts-bedrock/contracts/cannon/MIPS.sol]:
```suggestion
        // write: like Linux write syscall. Splits unaligned writes into aligned writes.
```

File: packages/contracts-bedrock/contracts/cannon/MIPS.sol [hunk 2]
```diff
@@ -140,27 +142,37 @@ contract MIPS {
...
-            // exit group
+        }
+        // exit group: Sets the Exited and ExitCode states to true and argument 0.
+        else if (syscall_no == 4246) {
             state.exited = true;
             state.exitCode = uint8(a0);
             return outputState();
-        } else if (syscall_no == 4003) { // read
+        }
+        // read: Similar behavior as Linux/MIPS with support for unaligned reads.
```
Comment (protolambda) [inline packages/contracts-bedrock/contracts/cannon/MIPS.sol]:
```suggestion
        // read: Like Linux read syscall. Splits unaligned reads into aligned reads.
```

File: packages/contracts-bedrock/contracts/cannon/MIPS.sol [hunk 3]
```diff
@@ -199,14 +224,19 @@ contract MIPS {
...
                 v0 = a2;
-            } else {
+            }
+            else {
                 v0 = 0xFFffFFff;
                 v1 = EBADF;
             }
-        } else if (syscall_no == 4055) { // fcntl
+        }
+        // fcntl: Similar behavior as Linux/MIPS.
```
Comment (protolambda) [inline packages/contracts-bedrock/contracts/cannon/MIPS.sol]:
```suggestion
        // fcntl: Like linux fcntl syscall, but only supports minimal file-descriptor control commands,
        // to retrieve the file-descriptor R/W flags.
```

**Review by protolambda — APPROVED** [review]
LGTM, just a few suggestions on comments.

**Review by clabby — APPROVED** [review]

**Pull request merged**